offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/StyledNotification.groovy

```diff
@@ -9,7 +9,7 @@ class StyledNotification extends Notification {
 
     StyledNotification(String caption) {
         super(caption)
-        setNotificationProperties()
+        setNotificationProperties(Type.HUMANIZED_MESSAGE)
```

Comment:
Unsure how this change is related to the intention of the PR.

Reply:
It is not directly. But I noticed it when analyzing the code and thought I would directly fix the implicit null here.